src/main/java/uk/gov/hmcts/reform/sendletter/controllers/ControllerResponseMessage.java

```diff
@@ -4,4 +4,5 @@
 final class ControllerResponseMessage {
 
     static final String RESPONSE_401 = "Invalid service authorisation header";
+    static final String RESPONSE_403 = "Service not configured";
```

Comment:
may be ,
"Service not authorised" 
"Service not configured to authorised" ?

Reply:
Status itself gives the information that service is not allowed to perform this action.

Reply:
Good point, `Service not authorised` might translate to our internal "not configured", but it would be ideal if the caller was told that they're just not authorised.